fyi_links.json

```diff
@@ -0,0 +1,6 @@
+{
+    "salary_link": "https://www.levels.fyi/company/Netflix/salaries/Software-Engineer/",
```

Comment:
Is this for testing or should there be a placeholder for the company name in the URL?

Reply:
It is for testing, we should probably align on a JSON format though that would be hosted on S3 before merging, any suggested formats would be greatly appreciated.